src/types.ts

```diff
@@ -160,6 +160,15 @@ export enum NftRevealType {
   WEEK = 'WEEK',
 }
 
+export enum AddressRestrictionType {
```

Comment:
Shouldn't this just be named `RestrictionType`? Why should it be narrowed down to Address?

Reply:
all restrictions are based on address, so wanted to make that clear here